services/static-webserver/client/source/class/osparc/theme/Appearance.js

```diff
@@ -251,6 +268,46 @@ qx.Theme.define("osparc.theme.Appearance", {
       }
     },
 
+    "pb-study/lock-status":  {
```

Comment:
About this. Makes no sense to me to inherit the appearance of its father component. But what do I know about QLoookxL

Reply:
this should now be resolved